backend/config/settings.py

```diff
@@ -33,6 +33,9 @@ def get_bool(key, default):
 SESSION_SAVE_EVERY_REQUEST = True
 SESSION_COOKIE_NAME = 'tsd_sessionid'
 
+# How long before HMAC signed URLs expire
+HMAC_SIGNED_URL_EXPIRATION_HOURS = 1
```

Comment:
We definitely can make expiration a lot longer. Some files, such as the URL embedded in email, may need to be accessed for a few days or even longer.

Reply:
My suggestion is to have a longer default, such as 2 weeks or 4 weeks. This expiration is essentially a measure against a brute-force method to crack the signature, which is probably highly unlikely given our use case (info is probably not valuable enough to attract professional hackers who can pull this off).